src/components/index-page/hero/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { StripesDecoration } from "@/app/conf/_design-system/stripes-decoration"
+import CheckIcon from "@/app/conf/_design-system/pixelarticons/check.svg?svgr"
+import { Button } from "@/app/conf/_design-system/button"
+import { ImageLoaded } from "@/app/conf/2025/components/image-loaded"
+
+import logoBlurred from "./logo-blurred.webp"
+import Head from "next/head"
+
+export function Hero() {
+  return (
+    <div className="relative bg-neu-50 dark:bg-neu-50/10">
+      <div className="gql-container flex flex-col-reverse lg:grid lg:grid-cols-2">
+        <div className="flex max-w-4xl flex-col justify-center gap-4 p-4 lg:min-h-[800px] xl:gap-8 xl:py-24 xl:pl-24 xl:pr-10">
+          <h1 className="typography-h1 max-w-3xl text-neu-900">
+            The query language for modern APIs
+          </h1>
+
+          <ul className="flex flex-col gap-2">
+            {[
+              "Deliver high-performance user experience at scale",
+              "Secure and stabilize your API with a strongly typed schema and validated queries",
+              "Reduce dependencies through efficient, distributed development",
```

Comment:
Can someone explain this one to me? What dependencies are we reducing?

Reply:
This is great, thanks @bignimbus.